### PR TITLE
Kafka-template

### PR DESCRIPTION
**Pull request by autumnust**
- simple kafka template 
- file detector running period (30s) added into gobblin/conf/gobblin-standalone.properties

### CODE REVIEW

**Comment by autumnust** [comment]
@ibuenros Could you review this template?

File: gobblin-runtime/src/main/java/gobblin/scheduler/PathAlterationListenerAdaptorForMonitor.java [hunk 1]
```diff
@@ -186,6 +186,10 @@ private void rescheduleJob(Properties jobProps)
     boolean runOnce = Boolean.valueOf(jobProps.getProperty(ConfigurationKeys.JOB_RUN_ONCE_KEY, "false"));
     // Reschedule the job with the new job configuration
     jobScheduler.scheduleJob(jobProps, runOnce ? new RunOnceJobListener() : new EmailNotificationJobListener());
+    LOG.info("[JobScheduler]"
```
Comment (ibuenros) [inline gobblin-runtime/src/main/java/gobblin/scheduler/PathAlterationListenerAdaptorForMonitor.java]: This looks like a debug message. If so, please use `LOG.debug`, also, why is it in multiple lines?

**Comment by ibuenros** [comment]
Just one comment, LGTM otherwise.

File: gobblin-runtime/src/main/java/gobblin/scheduler/PathAlterationListenerAdaptorForMonitor.java (hunk 1 above)
Comment (autumnust) [inline gobblin-runtime/src/main/java/gobblin/scheduler/PathAlterationListenerAdaptorForMonitor.java]: I guess IntelliJ's formatter doesn't like me :(

**Pull request merged**